src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ArchiveGoalTest.kt

```diff
@@ -78,6 +87,28 @@ class ArchiveGoalTest : IntegrationTestBase() {
           .hasArchiveReason(ReasonToArchiveGoal.OTHER)
           .hasArchiveReasonOther(reasonOther)
       }
+
+    await.untilAsserted {
```

Comment:
We can now assert that the Timeline Event was correctly recorded, and that the telemetry (App Insights) event was sent